packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -122,7 +122,7 @@ export class CdkErrorMapper {
     },
     {
       errorRegex: new RegExp(
-        `(SyntaxError|ReferenceError|TypeError):((?:.|${this.multiLineEolRegex})*?at .*)`
+        `(SyntaxError|ReferenceError|TypeError)( \\[[A-Z_]+])?:((?:.|${this.multiLineEolRegex})*?at .*)`
```

Comment:
if you wonder why `]` is not escaped.
![image](https://github.com/user-attachments/assets/901935c6-e9a3-4911-a03c-8d7aef7b3fe8)